apps/api/src/server/utils/application-factory-for-tests.js

```diff
@@ -9,39 +9,67 @@ const generateApplicationReference = () => {
 
 /**
  *
- * @param {{id: number, status: string, modifiedAt: Date}} arg
+ * @param {{
+ * 	id?: number,
+ * 	status?: string,
+ * 	modifiedAt?: Date,
+ * 	title?: string | null,
+ *  description?: string | null,
+ * 	regions?: object[],
+ *  subSectorId?: number | null,
+ * 	subSector?: import('@pins/api').Schema.SubSector,
+ * 	zoomLevelId?: number | null
+ * }} arg
  * @returns {import('@pins/api').Schema.Case}
  */
-export const applicationFactoryForTests = ({ id, status = 'open', modifiedAt = new Date() }) => {
+export const applicationFactoryForTests = ({
```

Comment:
would be nice to have common factories for BE and FE

Reply:
We can do!

Reply:
Except it is one thing to have a factory for the Database, and another to have a factory for the API responses for example. Not sure where you are using factories?

Reply:
for unit tests

Reply:
happy to have a chat about this at some point. absolutely not urgent